test/externalTests/ens.sh

```diff
@@ -28,22 +28,23 @@ verify_input "$@"
 BINARY_TYPE="$1"
 BINARY_PATH="$2"
 
-function compile_fn { npx truffle compile; }
-function test_fn { npm run test; }
+function compile_fn { yarn build; }
+function test_fn { yarn test; }
 
 function ens_test
 {
-    local repo="https://github.com/ensdomains/ens.git"
+    local repo="https://github.com/ensdomains/ens-contracts.git"
     local branch=master
-    local config_file="truffle.js"
+    local config_file="hardhat.config.js"
 
-    local compile_only_presets=()
+    local compile_only_presets=(
+        legacy-no-optimize        # Compiles but tests fail to deploy GovernorCompatibilityBravo (code too large).
+    )
     local settings_presets=(
         "${compile_only_presets[@]}"
-        #ir-no-optimize           # "YulException: Variable var_ttl_236 is 1 slot(s) too deep inside the stack."
-        #ir-optimize-evm-only     # "YulException: Variable var_ttl_236 is 1 slot(s) too deep inside the stack."
-        ir-optimize-evm+yul
-        legacy-no-optimize
+        #ir-no-optimize           # "YulException: Variable var__945 is 1 slot(s) too deep inside the stack."
+        #ir-optimize-evm-only     # "YulException: Variable var__945 is 1 slot(s) too deep inside the stack."
+        #ir-optimize-evm+yul      # "YulException: Variable _5 is 1 too deep in the stack [ _5 usr$i usr$h _7 usr$scratch usr$k usr$f _4 usr$len usr$j_2 RET _2 _1 var_data_mpos usr$totallen usr$x _12 ]"
```

Comment:
How is this to be read? `#<setting> # Description`? Or is it `#<setting>` causes `#YulException:...`?

Reply:
The latter. It means that with this particular configuration preset the project cannot be compiled (causes the compilation error in the comment). I'll add more text to make it clearer.

Reply:
Done.